frontend/src/Onboarding/OnboardingInfoScreen.tsx

```diff
@@ -84,7 +83,6 @@ type OnboardingScreenProps = OnboardingReduxStoreProps &
   OnboardingReduxDispatchProps;
 
 interface OnboardingScreenState {
-  year?: number;
```

Comment:
`year` was originally equivalent of `academicYear`, which no longer needs to be part of state here since academicYear should be in redux by now